x-pack/plugins/serverless_security/public/components/get_started/get_started.tsx

```diff
@@ -48,7 +61,18 @@ export const GetStartedComponent: React.FC = () => {
               padding-left: ${euiTheme.size.xs};
             `}
           >
-            <span>{GET_STARTED_PAGE_TITLE}</span>
+            <>
+              <span>{GET_STARTED_PAGE_TITLE}</span>
+              <span
+                role="img"
+                aria-label="emoji-wave"
+                css={css`
+                  padding: 0 ${euiTheme.size.xs};
+                `}
+              >
+                👋
```

Comment:
I was told that we should not use emojis in the code, can we create a SVG and use the `EuiIcon`?

Reply:
Sure, I'll remove this for now until I've got the image from UX.